lib/Capabilities.php

```diff
@@ -82,14 +83,17 @@ class Capabilities implements ICapability {
 	private $capabilitiesService;
 	/** @var PermissionManager */
 	private $permissionManager;
+	/** @var AppManager */
+	private $appManager;
 
 	private $capabilities = null;
 
-	public function __construct(IL10N $l10n, AppConfig $config, CapabilitiesService $capabilitiesService, PermissionManager $permissionManager) {
+	public function __construct(IL10N $l10n, AppConfig $config, CapabilitiesService $capabilitiesService, PermissionManager $permissionManager, AppManager $appManager) {
```

Comment:
```suggestion
	public function __construct(IL10N $l10n, AppConfig $config, CapabilitiesService $capabilitiesService, PermissionManager $permissionManager, IAppManager $appManager) {
```

Should use the IAppManager from the OCP namespace which is considered the public and stable API.

Reply:
Fixed! 

Also modified my include path in PhpStorm to just include the public API, should avoid this in the future :)